### DISCUSSION TITLE
Rumors about a expliot using  secure test browser

### DISCUSSION THREAD

**Discussion by U-Nazu** [3kh0/ext-remover#1004] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
One dude in my school found out a way using the testing browser for SBACs but it's gate keeped for 20$ I need info on this

**Comment by AshtonDavies** [comment]
[SKIOVOX](https://chrose.pages.dev/detail/SKIOVOX)?

**Reply by U-Nazu** [reply]
Not sure my close friend said it's in the sign out apps using and secure test browser and the dude who had it I was gonna pay him and he was "20K and payments" what a kobhead

**Reply by AshtonDavies** [reply, marked as answer]
Yeah, that ain't worth the payment, and based on that little info, it very well could be a scam for SKIOVOX, which is already public and free to use.

But if you can somehow get it and find out it's something else, I would like to see what this thing is all about.

**Reply by U-Nazu** [reply]
Tysm mate!

**Comment by v4pl** [comment]
Probely Skiovox but i don't belive Skiovox works on secure test broswer cause for me that doesn't work also your Chromebook is probley up to date and Skiovox got patched any way unless u downgrade like me

**Reply by AshtonDavies** [reply]
I think I remember hearing of it working on SecureTestBrowser, but I don't know for sure.

**Reply by py660** [reply]
I had in fact tried it on securetestbrowser, and it worked
